config/packages/test/doctrine.yaml

```diff
@@ -0,0 +1,4 @@
+doctrine:
+  dbal:
+    url: 'mysql://root@db:3306/basicrum_test'
```

Comment:
I had to use root here as it's dropping and creating DB in bootstrap.php and should have required permissions.